.github/workflows/docker-build-test.yaml

```diff
@@ -130,6 +130,23 @@ jobs:
   # This is a PR required job.
   rust-images:
     needs: [permission-check, determine-docker-build-metadata]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set IMAGE_TAG based on PR label or branch/environment
+      run: |
+        BRANCH_NAME=${GITHUB_REF#refs/heads/}
+        SHORT_SHA=$(echo ${{ needs.determine-docker-build-metadata.outputs.gitSha }} | cut -c1-7)
+        PR_LABELS=$(echo "${{ join(github.event.pull_request.labels.*.name, ' ') }}")
+
+        if [[ "$PR_LABELS" =~ CICD:build(-[a-zA-Z0-9]+)?-images ]]; then
```

Comment:
Instead of relying on hardcoded build labels, you should probably check for other cases and make ci a fallback / catch all tag